homeassistant/components/fritz/common.py

```diff
@@ -30,16 +30,18 @@
 from homeassistant.helpers import update_coordinator
 from homeassistant.helpers.device_registry import (
     CONNECTION_NETWORK_MAC,
-    async_entries_for_config_entry,
-    async_get,
+    async_entries_for_config_entry as async_entries_for_config_device_entry,
+    async_get as async_get_device_registry,
     format_mac,
 )
```

Comment:
```python
from homeassistant.helpers import device_registry as dr
```
than just use `dr.async_get()` and `dr.async_entries_for_config_entry()` ...

Reply:
Done